psp/models/recent_history.py

```diff
@@ -156,7 +161,9 @@ def __init__(
         nwp_dropout: Probability of removing the NWP data (replacing it with np.nan).
             This is only used at train-time.
         nwp_tolerance: How old should the NWP predictions be before we start ignoring them.
-            See `NwpDataSource.get`'s documentation for details..
+            See `NwpDataSource.get`'s documentation for details.
+        nwp_patch_size: Size of the patch to use for the NWP data. This is in degrees.
```

Comment:
Worth making a note in the PR somewhere that this is not completed yet (nwp_patch_size)

Reply:
Ill take it out for the moment, and ill add some others that are missing